ons_alpha/taxonomy/views.py

```diff
@@ -0,0 +1,63 @@
+# Create TopicViewSet
+from wagtail.admin.views.generic.chooser import ChooseResultsView, ChooseView
+from wagtail.admin.viewsets.chooser import ChooserViewSet
+from wagtail.admin.viewsets.model import ModelViewSet
+
+from .models import Topic
+
+
+class TopicChooserMixin:
+    def get_object_list(self):
+        # Only show top-level topics
+        return Topic.get_root_nodes()
+
+
+class SubTopicChooserMixin:
+    def get_object_list(self):
+        # Only show topic that have a parent
+        return Topic.objects.exclude(depth=1).order_by("path")
+
+
+class TopicChooseView(TopicChooserMixin, ChooseView): ...
+
+
+class TopicChooseResultsView(TopicChooserMixin, ChooseResultsView): ...
+
+
+class SubTopicChooseView(SubTopicChooserMixin, ChooseView): ...
+
+
+class SubTopicChooseResultsView(SubTopicChooserMixin, ChooseResultsView): ...
+
+
+class TopicViewSet(ModelViewSet):
+    model = Topic
+    icon = "tag"
+    add_to_admin_menu = True
+    menu_order = 200
+
+
+class TopicChooserViewSet(ChooserViewSet):
+    model = Topic
+
+    icon = "tag"
+    choose_one_text = "Choose a topic"
+    choose_another_text = "Choose another topic"
+    url_filter_parameters = ["country"]
+    preserve_url_parameters = ["multiple", "country"]
```

Comment:
these won't do anything as we don't have a country field.
This looks like a remnant from https://docs.wagtail.org/en/stable/extending/generic_views.html#limiting-choices-via-linked-fields

It should be possible to use the `linked_fields` param on the widget with a regex, on the panel definition on the Bulletin page, will check tomorrow

Reply:
Whoops, yup just remnants from testing. Removed.